### PR TITLE
App skeleton

### PR DESCRIPTION
**Pull request by jacobvjk**
This PR:

- sets up the project
- adds `.DS_Store` to `.gitignore`
- adds a very basic shiny app skeleton that allows uploading a file (later: portfolio) and downloading the processed file (currently the same as the input, later the portfolio matched to NACE sectors and aggregated to that level)

Open:
- It is currently locally hosted (to be moved to shinyapps.io)
- It does not yet contain any of its final data prep and wrangling functionality (to be adapted from StressTestingModelDev scripts)

### CODE REVIEW

**Comment by jacobvjk** [comment]
@jdhoffa @cjyetman take this as a suggestion for the conversation we are about to have.

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,10 @@ vignettes/*.pdf
 
 # R Environment Variables
 .Renviron
+
+### MacOS specific ###
+# Folder view configuration files
+# Mac files
+.DS_STORE
+**/.DS_STORE
```
Comment (cjyetman) [inline .gitignore:45]: pretty sure this second ds_store thing is not necessary... the first one captures a file named the in any folder, no?

**Review by cjyetman — APPROVED** [review]

**Comment by jdhoffa** [comment]
FYI, working from a new repo now!

**Comment by jdhoffa** [comment]
https://github.com/2DegreesInvesting/r2dii.nace.matching

**Pull request merged**